packages/drafts-realm/country.gts

```diff
@@ -0,0 +1,25 @@
+import {
+  contains,
+  field,
+  Component,
+  CardDef,
+} from 'https://cardstack.com/base/card-api';
+import StringField from 'https://cardstack.com/base/string';
+
+export class Country extends CardDef {
+  static displayName = 'Country';
+  @field name = contains(StringField); // required
```

Comment:
What does `// required` mean here?

Reply:
I copied it from another existing card. I think it is just a note to inform you that field is required.